### DISCUSSION TITLE
Destro legendary

### DISCUSSION THREAD

**Discussion by marekCs** [simulationcraft/simc#5820] [Ideas]
Hello I would like to see Destro warlock with the Cinders legendary instead of the Relic one. I’m pretty sure the damage will be better.

**Comment by scamille** [comment]
cc @simulationcraft/warlock 

Feel free to try this yourself, and post your findings as raidbot links, one for each setup.

**Reply by marekCs** [reply]
ok I was simmed it and I got a slightly better dps with Cinders legendary: https://www.raidbots.com/simbot/report/wG85nirevx4iha9RXeAn6C
